plugins/services/src/js/structs/ServiceTree.js

```diff
@@ -48,7 +50,19 @@ module.exports = class ServiceTree extends Tree {
       }
 
       // Create the appropriate service according to definition
-      return ServiceUtil.createServiceFromResponse(item);
+      if (ServiceValidatorUtil.isPodResponse(item)) {
+        return new Pod(item);
+      }
+
+      if (ServiceValidatorUtil.isFrameworkResponse(item)) {
+        return new Framework(item);
+      }
+
+      if (ServiceValidatorUtil.isApplicationResponse(item)) {
+        return new Application(item);
+      }
+
+      throw Error("Unknown service response: " + JSON.stringify(item));
```

Comment:
Is this error caught anywhere?

Reply:
no idea, just copy&pasted this code, will check 👍

Reply:
Okay, do we want that error to bubble through?

Reply:
I think in that case its okay, if we cant identify something as Pod/Application/Framework, we have a problem 🤔